Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -301,9 +310,10 @@ extension String {
     /// Returns a copy of `self` with the leading whitespace common in each line removed.
     public func stringByRemovingCommonLeadingWhitespaceFromLines() -> String {
         var minLeadingWhitespace = Int.max
+
         enumerateLines { line, _ in
-            let lineLeadingWhitespace = line.countOfLeadingCharactersInSet(whitespaceAndNewlineCharacterSet)
-            if lineLeadingWhitespace < minLeadingWhitespace && lineLeadingWhitespace != line.characters.count {
```

Comment:
That check makes it so whitespace-only lines aren't considered to detect the horizontal offset of the comment body. For example:

```swift
    /**
        Comment starts here

        Previous line has no trailing whitespace (just a newline), but isn't where the comment offset starts.
    */
```

But I'd expect adding the asterisk character to the character set would still work with this scheme.